### ISSUE TITLE
#19242 breaks compatibility of command line args on kube-controller-manager

### ISSUE THREAD

**Issue by mikedanese** [kubernetes/kubernetes#26002]
Not specifying --service-cluster-ip-range added in #19242 causes a panic in nodecontroller which causes controller-manager to crash loop.

This feels like a v1.3 blocker.

Ref #19242

@bgrant0607 @mhrgoog @mqliang @wojtek-t @gmarek @lavalamp @goltermann @davidopp

**Comment by mikedanese** [comment]
```
 I0520 22:21:43.539213    2933 plugins.go:71] No cloud provider specified.
 I0520 22:21:43.539397    2933 nodecontroller.go:157] Sending events to apiserver.
 panic: runtime error: invalid memory address or nil pointer dereference [signal 0xb code=0x1 addr=0x0 pc=0x7f28ab]
 goroutine 1 [running]:
 panic(0x2303e40, 0xc82000e100)
         /usr/local/go/src/runtime/panic.go:481 +0x3e6

k8s.io/kubernetes/pkg/controller/node.(*NodeController).filterOutServiceRange(0xc820144000)
         /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/node/nodecontroller.go:342+0x2b
 k8s.io/kubernetes/pkg/controller/node.(*NodeController).Run(0xc820144000,0x2540be400)
```

**Comment by mqliang** [comment]
Sorry, I am not familiar with the code under /cluster and /test.

**Comment by bgrant0607** [comment]
@mqliang @gmarek @wojtek-t Component flags constitute an API, with similar compatibility rules. We can't add new required flags to binaries. That breaks all existing Kubernetes cluster deployment code.

**Comment by mikedanese** [comment]
@mqliang no worries. We should have caught these errors in the PR builder. It's an item to increase coverage in pre-submit PR checks

**Comment by zmerlynn** [comment]
Sorry, I should have caught this in review, too. This flag should default to empty if it's not specified, it's a relatively easy fix.

**Comment by gmarek** [comment]
Fixed by #26054
